src/Business/Camera/Common/Repositories/ICameraRepository.cs

```diff
@@ -6,6 +6,6 @@ namespace Business.Camera.Common.Repositories
 
     public interface ICameraRepository
     {
-        public IEnumerable<MCamera> Insert(IEnumerable<MRegisterCamera> input);
+        public MCamera Insert(MRegisterCamera input);
```

Comment:
Should we maintain the old way of insertion? By keeping an array of inputs? It's gonna be used by camera seeds also.

Reply:
done